ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -273,3 +336,49 @@ func mergeJsonCommonKey(mergeWith, mergeTo, key string) (string, error) {
 	}
 	return string(newJson), nil
 }
+
+func applyWorkloadPinningInstallConfigOverrides(clusterSpec *Clusters) (result string, err error) {
```

Comment:
As far as I understand this func to replace the use of  [getWorkloadManifest](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go#L333)  is that correct ? if yes we need to remove the machineConfig OR the machineConfig for workload will till remain ?

Reply:
The siteConfig tool (binary) can be used to deploy clusters on prior releases as well. This new feature shows up in 4.13, so if the user is deploying a 4.12 cluster they will need the MachineConfig, but if they are deploying 4.13 they need this annotation (and do not need the MachineConfig)

Reply:
I believe that if the user leaves "cpuset" out of their config we will automatically leave out the workload partitioning manifests. We will need to update the documentation to note that in 4.13+ the user specifies cpuPartitioningMode and not cpuset.